server/macros/__init__.py

```diff
@@ -4,5 +4,22 @@
 
 Use `superdesk.macros.macros.register` for registration.
 """
+import sys
+import imp
+macros = ['macros.currency', 'macros.abstract_populator']
 
-from . import currency  # noqa
+try:
```

Comment:
What I observed is that, we need to force reimporting the modules in a package, otherwise the package does not reflect the changes to the macros. It does add the new ones, because we would have a new import statement in __init__.py for the new macro. So (as it normally does) once a module is loaded it doesn't load again if it is not forced for reload.